File: scala-stuff/src/main/scala/io/github/flyingcowmoomoo/arrowcatsfpguild/scala/Interoperability.scala
package io.github.flyingcowmoomoo.arrowcatsfpguild.scala

import io.github.flyingcowmoomoo.arrowcatsfpguild.java.MyApi


object Interoperability {

  import scala.jdk.CollectionConverters._
  val api = new MyApi()
  api.setNumbers(List(1, 2, 3).map(Integer.valueOf).asJava)

}

File: kotlin-stuff/src/main/kotlin/io/github/flyingcowmoomoo/kotlin/arrowcatsfpguild/Interoperability.kt
package io.github.flyingcowmoomoo.kotlin.arrowcatsfpguild

import io.github.flyingcowmoomoo.arrowcatsfpguild.java.MyApi

object Interoperability {


    fun doThings() {

        // Kotlin
        val api = MyApi()
        api.addNumbers(listOf(1, 2, 3))
    }


}